nano/node/cli.cpp

```diff
@@ -711,6 +712,31 @@ std::error_code nano::handle_node_options (boost::program_options::variables_map
 			}
 		}
 	}
+	else if (vm.count ("update_config"))
+	{
+		nano::network_params network_params{ nano::network_constants::active_network };
+		nano::tomlconfig default_toml;
+		nano::tomlconfig current_toml;
+		nano::daemon_config default_config{ data_path, network_params };
+		nano::daemon_config current_config{ data_path, network_params };
+
+		std::vector<std::string> config_overrides;
+		auto error = nano::read_node_config_toml (data_path, current_config, config_overrides);
+		if (error)
+		{
+			std::cerr << "Could not read existing config file\n";
+			ec = nano::error_cli::invalid_arguments;
```

Comment:
I think the error `nano::error_cli::reading_config` is more appropriate here.

Reply:
Agreed. I'll change it